foundationdb/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,53 @@
+name: FoundationDB
+files:
+- name: foundationdb.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: cluster_file
+      description: |
+        A path to the cluster file specifing connection to the FoundationDB.
+        If it is not passed, the check will rely on the binding library to find
+        the file using standard locations (e.g. `/etc/foundationdb/fdb.cluster` or
+        `/etc/foundationdb/fdb.cluster`).
+      value:
+        example: /var/lib/foundationdb/fdb.cluster
+        type: string
+    - name: tls_certificate_file
```

Comment:
Have you considered using our existing `http` template? We already have these config options in it: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml#L240

Reply:
Using in what way? You mean matching the same keys? That'd be one consistency (across Datadog integrations); we've instead gone with consistency with how these options are named in FoundationDB.